tests/test_full.yml

```diff
@@ -64,11 +64,11 @@
     - path: results_test/pathway_analysis/DE_contrast_condition_genotype_WT_vs_KO/gost_pathway_gostplot.pdf
     - path: results_test/pathway_analysis/DE_contrast_condition_genotype_WT_vs_KO/gost_pathway_gostplot.png
     - path: results_test/pathway_analysis/DE_contrast_condition_genotype_WT_vs_KO/pathway_enrichment_results.tsv
-      md5sum: 21b907d52008f66f9a85190400fd211e
+      # md5sum: 21b907d52008f66f9a85190400fd211e
```

Comment:
Md5sums were not stable for the pathway results for some test profiles. I suspect the reason for this is that gprofiler has been updated with new data from Ensembl. Therefore the results [at this line](https://github.com/qbic-pipelines/rnadeseq/blob/e53bbae31382cecfcf2d29f808be84ed5b235a19/assets/RNAseq_report.Rmd#L2029) are probably not reproducible. Interestingly, the link is misformatted, but works.
This reproducibility issue is also described in the [FAQ](https://biit.cs.ut.ee/gprofiler/page/faq). One solution could be to select a specific archived version: https://biit.cs.ut.ee/gprofiler/page/archives/

Reply:
If that pipeline would have continued support, I would suggest for controlling the database version. However, because other pipelines should eventually succeed this one, I'd recommend to keep it here as is but to make sure the next workflow does that right.